msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -176,7 +765,9 @@ To reconfigure the bot, follow these steps:
 
    :::image type="content" source="../../assets/images/bots/reconfigure-bot-settings.png" alt-text="Screenshot shows the Adaptive Card with settings icon to reconfigure.":::
 
-   The bot sends a response message.
+   The bot sends a response message and now the bot is configured.
+
+   :::image type="content" source="../../assets/images/bots/setting-up-bot.png" alt-text="Screenshot shows the bot configuration set up is complete.":::   
 
 ## Code sample
```

Comment:
Can you add a best-practices section here with the bots team recommendations for how to best use this feature?

Sid mentioned things like:
- bot should explain how to reconfigure the app in the welcome message
- others???